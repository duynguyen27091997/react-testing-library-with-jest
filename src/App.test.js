import React from 'react'
import {render, cleanup} from '@testing-library/react'
import App from './App'

afterEach(cleanup)

test('should take a snapshot again', () => {
    const { asFragment } = render(<App />)

    expect(asFragment()).toMatchSnapshot()
})