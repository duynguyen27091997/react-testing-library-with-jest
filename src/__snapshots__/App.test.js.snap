// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot again 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <h1>
      Testing Updated 1
    </h1>
  </div>
</DocumentFragment>
`;
